lib/zendesk_apps_tools/command.rb

```diff
@@ -33,6 +33,10 @@ class Command < Thor
                        default: false,
                        hide: true,
                        desc: 'Create a version 1 app template (Deprecated)'
+    method_option :scaffold, type: :boolean,
+                       default: false,
+                       hide: true,
+                       desc: 'Create a version 2 app template with latest scaffold'
```

Comment:
😄 

just perhaps for indentation and consistency,
👉 default 
👉 hide
👉 desc

Reply:
referring to line 28 and 32's option parameters

Reply:
👍